backend/app/Http/Controllers/TransactionController.php

```diff
@@ -11,6 +11,38 @@
 
 class TransactionController extends Controller
 {
+    public static function creditTransaction(Account $account, $payload)
```

Comment:
refactor so that the api endpoint functions show first, 
ex

index
show

..
creditTransaction
transferTransaction